lib/internal/webstreams/readablestream.js

```diff
@@ -38,9 +38,7 @@ const {
   },
 } = require('internal/errors');
 
-const {
-  DOMException,
-} = internalBinding('messaging');
+const { DOMException } = internalBinding('messaging');
```

Comment:
Unrelated style changes. Please leave the require statements formatted as they are.